server/app/files.go

```diff
@@ -120,3 +123,18 @@ func (a *App) GetFileReader(teamID, rootID, filename string) (filestore.ReadClos
 
 	return reader, nil
 }
+
+func (a *App) MoveFile(channelID, teamID, boardID, filename string) error {
+	oldPath := filepath.Join(channelID, boardID, filename)
+	newPath := filepath.Join(teamID, boardID, filename)
+	err := a.filesBackend.MoveFile(oldPath, newPath)
+	if err != nil {
+		a.logger.Error("ERROR moving file",
```

Comment:
would it be useful to do a `logger.Debug` too e.g. the file was successfully moved (`err == nil`)?

Reply:
May-be, I went back and forth and decided not?